cartridges/plugin_gtm/cartridge/controllers/GTM.js

```diff
@@ -6,9 +6,9 @@ var gtmHelpers = require('*/cartridge/scripts/gtm/gtmHelpers');
 
 server.get('CustomerData', function (req, res, next) {
     var customerData = gtmHelpers.getCustomerData(req);
-    
+
     res.render('/gtm/gtmCustomerData', {
-        customerData: JSON.stringify(customerData)
+        customerData: JSON.stringify(customerData),
```

Comment:
Can you remove all these trailing commas please.

Reply:
I undid all the code formatting/linting